package.json

```diff
@@ -11,21 +11,20 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "dev": "webpack-dev-server --inline --config ./build/webpack.dev.conf.js",
+    "dev": "vue-cli-service serve",
+    "build": "yarn theo && vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit",
     "start": "npm-run-all --parallel styleguide theo:onchange theo dev",
     "unit": "jest --config ./test/unit/jest.conf.js --coverage",
-    "build": "npm-run-all theo node:build",
-    "build:app": "npm run build",
-    "build:system": "npm-run-all theo node:build:system",
-    "build:docs": "npm-run-all theo styleguide:build",
-    "lint": "eslint '**/*.{js,vue}' --cache",
-    "node:build": "node ./build/build.js",
-    "node:build:system": "node ./build/build-system.js",
-    "styleguide": "vue-styleguidist server --open --config ./config/docs.config.js",
-    "styleguide:build": "vue-styleguidist build --config ./config/docs.config.js",
+    "build:app": "yarn build",
+    "build:system": "yarn build --target lib ./src/system.js",
+    "build:docs": "yarn theo && yarn styleguide:build",
+    "styleguide": "vue-cli-service styleguidist --config ./config/docs.config.js",
+    "styleguide:build": "vue-cli-service styleguidist:build --config ./config/docs.config.js",
     "theo": "theo ./src/tokens/tokens.yml --transform web --format map.scss,scss,raw.json,json --dest ./src/assets/tokens",
-    "theo:onchange": "onchange \"./src/tokens/*.yml\" -- npm run theo",
-    "test": "npm-run-all theo unit",
+    "theo:onchange": "onchange \"./src/tokens/*.yml\" -- yarn theo",
+    "test": "yarn theo && yarn unit",
```

Comment:
I don’t think && will work in windows which was the original reason for switching to npm-run-all. I think we should still support both windows & macOS in addition to Linux users.

Reply:
I just tried it on windows and I believe it works. We can try it together if you wish.

Reply:
It depends how the user has set up their system. I’m 99% sure it won’t work for some users as there even was previously a bug about this in vueds that I had to fix (by getting rid of &’s). It’s also mentioned in multiple other places too like f.ex. npm-run-all’s readme.